### ISSUE TITLE
Integrating react-native-audio

### ISSUE THREAD

**Issue by jsierles** [zmxv/react-native-sound#49]
Hey, I'm the author of [react-native-audio](https://github.com/jsierles/react-native-audio).

After some discussion here: https://github.com/jsierles/react-native-audio/issues/74#issuecomment-229270324, it seems like a good idea to integrate these two libraries.

Would you be interested? If so, I can look at a PR to add iOS recording support here.

**Comment by zmxv** [comment]
Hi Joshua, I think it's a good idea to merge the libraries. Do you plan to add Android support soon?

**Comment by jsierles** [comment]
Yeah android support is on my list, and it would be easier to focus on that than on my poor playback support. I'll look into a PR for this!
